.github/pull_request_template.md

```diff
@@ -32,7 +32,7 @@ Please delete any sections below which you do not need to fill out. You may keep
 <summary> Expand for Help </summary>
 
 - Have questions about the review or deployment process? View our [contributing docs](https://github.com/the-hideout/tarkov-dev/blob/main/CONTRIBUTING.md)
-- Need additional help and want to chat in real time? Join our [community Discord](https://discord.gg/XPAsKGHSzH)
+- Need additional help and want to chat in real time? Join our [community Discord](https://discord.gg/VGq6xtTD)
```

Comment:
Can you explain a bit more why? Seems like they both serve the same purpose unless strictly for just analytics / tracking of users who have joined from the new link? (Not against it just would be interested in why its relevant)